tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -107,7 +105,8 @@ function prepareFunction(type = 'standard', appName = 'commerce') {
           .replace('%%APP_NAME%%', appName));
     default:
       return k8s.loadAllYaml(functionYaml.toString()
-          .replace('%%URL%%', 'findEnv("GATEWAY_URL") + "/site/orders/" + code'));
+          .replace('%%URL%%', '"http://central-application-gateway.kyma-system:8080' +
+                   '/commerce/sap-commerce-cloud-commerce-webservices/site/orders/" + code'));
```

Comment:
Hey, there is no difference between this one and the `central-app-gateway` case. Maybe we can just make it the default one? 
I assume there is no longer an option to use the old Application Gateway, right?

Reply:
Or just make the `central-app-gateway` the default case, good point